synapse/handlers/sync.py

```diff
@@ -695,7 +725,7 @@ def current_sync_for_user(self, sync_config, since_token=None, full_state=False)
 
         return summary
 
-    def get_lazy_loaded_members_cache(self, cache_key):
+    def get_lazy_loaded_members_cache(self, cache_key: Any) -> LruCache:
```

Comment:
given that it adds the key to the cache, we probably need to assert the type of the key

```suggestion
    def get_lazy_loaded_members_cache(self, cache_key: Tuple[str,str]) -> LruCache:
```